chibuezeayogu/File-Transfer-App
# #164676799 Implement view document

**What does the PR do?**
- This PR allows logged in users to view an uploaded document/file

**Description of Task to be completed?**
- Logged in users are able to view uploaded document/file

**How should this be manually tested?**
- Clone the repository [File-Transfer-App](https://github.com/chibuezeayogu/File-Transfer-App.git) and
- Checkout to  this branch `ft-implement-view-document-164676799`
- run `bundle install` then `rails s` on the terminal to start the server
- navigate to `http://localhost:3000/document/id` to view an uploaded document
- run`rspec` on the terminal and ensure all test are passing

**What are the relevant pivotal tracker stories?**
- [#164676799](https://www.pivotaltracker.com/story/show/164676799)

**Screenshots**
- N/A
